version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.0.45-SNAPSHOT"
+ThisBuild / version := "1.0.0-SNAPSHOT"
```

Comment:
It will not be published with `-SNAPSHOT` on the end, this is just necessary for the GHA workflow.